plugins/woocommerce/includes/wc-order-functions.php

```diff
@@ -934,14 +934,27 @@ function wc_update_coupon_usage_counts( $order_id ) {
 	}
 
 	$has_recorded = $order->get_data_store()->get_recorded_coupon_usage_counts( $order );
+	$invalid_statuses = array( 'cancelled', 'failed', 'trash' );
```

Comment:
```suggestion
	$has_recorded     = $order->get_data_store()->get_recorded_coupon_usage_counts( $order );
	$invalid_statuses = array( 'cancelled', 'failed', 'trash' );
```

Appease the code style linter